pom.xml

```diff
@@ -0,0 +1,171 @@
+            </plugin>
+
+
+            <plugin>
+                <groupId>com.atlassian.maven.plugins</groupId>
+                <artifactId>bash-maven-plugin</artifactId>
+                <version>1.0-ts-20141204</version>
+                <executions>
+                    <execution>
+                        <id>package</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <script>
+                                rsync -a --delete ${project.basedir}/docker/ ${project.build.directory}/docker
+                                cp -p ${project.build.directory}/${project.build.finalName}.war ${project.build.directory}/docker/root.war
+                                find ${project.build.directory}/docker | xargs touch -t 197409101630.00
+                                docker build -t tradeshift/${docker.imageName} ${project.build.directory}/docker
+                            </script>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>deploy</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>deploy</phase>
+                        <configuration>
+                            <script>
+                                <!--docker push ${docker.imageName}-->
+                                echo '123'
+                            </script>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
+
+        <!-- Jersey -->
+        <dependency>
+            <groupId>org.glassfish.jersey.core</groupId>
+            <artifactId>jersey-server</artifactId>
+            <version>${jersey.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.glassfish.jersey.media</groupId>
+            <artifactId>jersey-media-moxy</artifactId>
+            <version>${jersey.version}</version>
+        </dependency>
+
+
+        <dependency>
+            <groupId>org.glassfish.jersey.containers</groupId>
+            <!-- if your container implements Servlet API older than 3.0, use "jersey-container-servlet-core" -->
+            <artifactId>jersey-container-servlet</artifactId>
+            <version>${jersey.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.core</groupId>
+            <artifactId>jersey-client</artifactId>
+            <version>${jersey.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.test-framework.providers</groupId>
+            <artifactId>jersey-test-framework-provider-inmemory</artifactId>
```

Comment:
Are you actually using this? If so I assume it should be under a test scope since it's a test framework.